src/main/resources/db/migration/V1_95__add_new_scheme_permissions_table.sql

```diff
@@ -0,0 +1,18 @@
+-- Table: public.scheme_permissions
+
+DROP TABLE IF EXISTS public.scheme_permissions;
+
+CREATE TABLE scheme_permissions (
```

Comment:
This isn't a great name for an intersection table with only 2 columns. `scheme_editors`, `scheme_admins`, `scheme_users` all seem more appropriate for what we're trying to use it for?

Reply:
yeah, changed it to grant_editors, no longer including the permissions column